crates/trigger/src/cli.rs

```diff
@@ -18,6 +18,10 @@ pub const FOLLOW_LOG_OPT: &str = "FOLLOW_ID";
 pub const WASMTIME_CACHE_FILE: &str = "WASMTIME_CACHE_FILE";
 pub const RUNTIME_CONFIG_FILE: &str = "RUNTIME_CONFIG_FILE";
 
+// pub const DISABLE_POOLING_ALLOCATOR: &str = "DISABLE_POOLING_ALLOCATOR";
+// pub const POOLING_SLOTS: &str = "POOLING_SLOTS";
+// pub const POOLING_: &str = "POOLING_";
```

Comment:
Are the comments here needed?

Reply:
Nope, thanks! Those were left over from a previous approach to making this configurable.